ffig/FFIG.py

```diff
@@ -66,20 +66,45 @@ def get_template_output(class_name, template_name):
 
 # -- END Old code --
 
+
+def write_bindings_to_disk(api_classes, env, args, output_dir):
+    """ Input:
+    - api_classes
+    - environment to get templates from 
+    - args
+    - output_dir where to write to
+    write the bindings to disk, return Nothing """
+    for binding in args.bindings:
+        with open(os.path.join(output_dir, get_template_output(args.module_name, get_template_name(binding))), "w") as output_file:
+            template = env.get_template(binding)
+            s = render_api_and_obj_classes(api_classes, template)
+            output_file.write(s)
+
+def build_model_from_source(inp, module_name):
```

Comment:
Assuming this is `inp` because `input` is a built in function? Maybe call it `input_file` or something?